playbooks/provisioning/openstack/sample-inventory/group_vars/OSEv3.yml

```diff
@@ -5,6 +5,10 @@ openshift_release: 1.5.1
 #openshift_release: v3.5
 openshift_master_default_subdomain: "apps.{{ env_id }}.{{ public_dns_domain }}"
 
+openshift_master_cluster_method: native
+openshift_master_cluster_hostname: "{{ groups.lb.0|default(groups.masters.0) }}"
```

Comment:
IIUC, these words
```
#For installation the value of
# openshift_master_cluster_hostname must resolve to the load balancer
# or to one or all of the masters defined in the inventory if no load
# balancer is present.
```
mean the `openshift_master_cluster_(public_)hostname` should be the cluster FQDN (`stack_name`) in fact, which resolves into the lb[0] for multi-master, or defaults to the masters[0] otherwise. Am I right, @tomassedovic ?..